# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/ExternalHelperAppChild.h
# Commit: baffa8a7b961
# Full Hash: baffa8a7b9618e9a02e4bd808adb71dead99cce5
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-11-12 09:43:07
# Regressor Bug: 1589270
# File Overlap Count: 1
# Description:
#   Bug 1589270 - Part 3: Convert nsExternalHelperApp to use BrowsingContext instead of nsIInterfaceRequestor. r=bzbarsky
#   
#   This also converts MaybeCloseWindowHelper, and results in the window close operations being always run in the parent (even without DocumentChannel).
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49528
# ==============================================================================

diff -r 68bc2ae9b89a -r baffa8a7b961 uriloader/exthandler/ExternalHelperAppChild.h
--- a/uriloader/exthandler/ExternalHelperAppChild.h	Fri Nov 08 03:01:03 2019 +0000
+++ b/uriloader/exthandler/ExternalHelperAppChild.h	Fri Nov 08 04:35:05 2019 +0000
@@ -36,8 +36,7 @@
  private:
   virtual ~ExternalHelperAppChild();
   MOZ_MUST_USE nsresult DivertToParent(nsIDivertableChannel* divertable,
-                                       nsIRequest* request,
-                                       BrowserChild* browserChild);
+                                       nsIRequest* request);
 
   RefPtr<nsExternalAppHandler> mHandler;
   nsresult mStatus;